__test__/lib/logger.test.js

```diff
@@ -0,0 +1,32 @@
+const loggerMiddleware = require('../../lib/logger.js');
+
```

Comment:
Hey Chris! It looks like that this is being required in wrong, which means that the tests aren't working.